packages/ava-react/src/NarrativeTextVis/line-charts/hooks/useSvgWrapper.tsx

```diff
@@ -1,17 +1,22 @@
 import React, { useRef, useState, useLayoutEffect } from 'react';
-import { getElementFontSize, DEFAULT_FONT_SIZE } from './getElementFontSize';
 
-type SvgReactFC = React.FC<React.SVGProps<SVGSVGElement>>;
+import { seedToken } from '../../theme';
+
+import { getElementFontSize } from './getElementFontSize';
+
+import type { PropsWithChildren } from 'react';
+
+type SvgProps = PropsWithChildren<React.SVGProps<SVGSVGElement>>;
```

Comment:
Because the React.FC type no longer includes the children props

Reply:
knowledge++